src/main/resources/application.yml

```diff
@@ -178,3 +178,5 @@ hmpps.sqs:
       errorVisibilityTimeout: 5
     courtsentencing:
       errorVisibilityTimeout: 1
+    contactperson:
+      errorVisibilityTimeout: 5
```

Comment:
fully expecting a create contact event to fail since the person event gets fired at same time - so a bit of breathing space before next retry